src/quartz/paths/org.yml

```diff
@@ -44,7 +44,7 @@ delete:
       content:
         application/json:
           schema:
-            $ref: "../schemas/Organization.yml"
+            type: string
```

Comment:
Grace and I decided to update the spec to match the existing Quartz behavior instead of updating Quartz itself. @randycoulman does this approach seem reasonable? I'm going to add a topic to the next tech discussion for what we think the command endpoints should return by default (entity being acted on vs success code), but wanted to at least make them match for now.

Reply:
This looks like a bug in the spec; we shouldn't be returning the deleted org, IMO.